Files/Interacts/Interaction.cs

```diff
@@ -613,6 +613,30 @@ private async void OpenSelectedItems(bool displayApplicationPicker)
             }
         }
 
+        internal void Prepare(string itemPath)
+        {
+            SetDirectory(itemPath);
+        }
+
+        private async void SetDirectory(string path)
+        {
```

Comment:
This method seems kind of redundant and probably doesn't belong in Interaction.cs. Does using/improving upon what exists in SetWorkingDirectoryAsync() solve the problem?